### PR TITLE
docs: postman collection

### PR DESCRIPTION
**Pull request by vinckr**
I have written a small guide for the Postman Collections, with some info on pitfalls. 
Maybe this could be a dedicated page in reference? But I felt it was not enough content to justify 4 separate pages and we also don't maintain this actively.
(I have a hook to our github repos in place, so it should update automatically. But I dont use Postman regularly, so I might miss when something breaks)

Let me know what you think!

PS:
I had to run 
`npm install  react-toggle`
for the docs to build correctly. 

Should I update the `package-lock.json` as well?

### CODE REVIEW

**Comment by aeneasr** [comment]
> PS:
> I had to run
> npm install react-toggle
> for the docs to build correctly.

That's really strange...did you run a very old copy of this repo?

**Comment by aeneasr** [comment]
> PS:
> I had to run
> npm install react-toggle
> for the docs to build correctly.

That's really strange...did you run a very old copy of this repo?

**Comment by vinckr** [comment]
> That's really strange...did you run a very old copy of this repo?

Probably it was probably some error on my end.
 If I checkout my fork now everything works as it should, so disregard my PS...

File: docs/docs/ecosystem/community.md [hunk 1]
```diff
@@ -8,61 +8,111 @@ Sponsors, contributors, and adopters make this ecosystem possible and shape it
 in significant ways.
 
 Important community resources include [GitHub](https://github.com/Ory),
-[Slack (chat)](https://slack.Ory.sh/) and GitHub Discussions for
+[Ory Chat](https://slack.Ory.sh/) and GitHub Discussions for
```
Comment (aeneasr) [inline docs/docs/ecosystem/community.md:11]: For most people, Slack is a clear indication of what to expect while "Ory Chat" is ominous. Maybe "Ory Slack Channel"?

File: docs/docs/ecosystem/community.md [hunk 2]
```diff
@@ -8,61 +8,111 @@ Sponsors, contributors, and adopters make this ecosystem possible and shape it
...
 [Ory Kratos](https://github.com/ory/kratos/discussions),
 [Ory Hydra](https://github.com/ory/hydra/discussions),
 [Ory Keto](https://github.com/ory/oathkeeper/discussions),
 [Ory Oathkeeper](https://github.com/ory/oathkeeper/discussions) and all other
 [Ory projects](https://github.com/ory/meta/discussions), you can also visit our
-archived [forum](https://community.Ory.sh/).
+archived old [forum](https://community.Ory.sh/).
```
Comment (aeneasr) [inline docs/docs/ecosystem/community.md:17]: Looks like all `ory` got replaced with `Ory` introduced by [this commit](https://github.com/ory/docs/commit/b884b68a8bfc86e6d9f2746eed78ca8cb28df67b). IntelliJ has a setting which respects casing. It's also a good idea to make the replacements case sensitive.

Can you please create another PR where the domain names are properly cased?

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: docs/docs/ecosystem/community.md (hunk 2 above)
Comment (vinckr) [reply docs/docs/ecosystem/community.md:17]: Sure, my bad!

**Pull request closed without merging**